### PR TITLE
add component Avatar and Image

### PR DESCRIPTION
**Pull request by Felipesdeveloper**
## Description

Avatars can be created by passing a img for prop `src` or without `src` for a avatar with color.

## How this was tested?

captura de tela 2019-03-07 as 17 40 12

## Type of changes

- [ ] Bug fix
- [x] New feature
- [ ] Breaking change

### CODE REVIEW

File: src/components/Avatar/Avatar.stories.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { ThemeProvider } from 'styled-components';
+import { storiesOf } from '@storybook/react';
+import { text } from '@storybook/addon-knobs';
+import { withDocs } from 'storybook-readme';
+
+import Avatar from './index';
+import createTheme from '../../styles/theme';
+import markdown from './README.md';
```
Comment (r0nho) [inline src/components/Avatar/Avatar.stories.js]: Vamos seguir um padrão, vamos utilizar o nome `readme` ou  `README` ao invés de markdown, o que acha?

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+## How this was tested?
+
+## Screenshots (if necessary)
+
+## Type of changes
+
+- [ ] Bug fix
+- [ ] New feature
+- [ ] Breaking change
```
Comment (r0nho) [inline .github/pull_request_template.md]: Boa! 👍

File: src/components/Avatar/Avatar.jsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+const Avatar = (props) => {
+  const { alt, src, ...attributes } = props;
+
+  if (src) {
+    return (
+      <StyledAvatar {...attributes} >
+        <StyledAvatar.Image alt={alt} src={src} />
+      </StyledAvatar>
+    );
+  }
```
Comment (williamespindola) [inline src/components/Avatar/Avatar.jsx]: I think this condition means that this component is too smart. What would be the case where this Avatar will be rendered without image?

Comment (williamespindola) [reply src/components/Avatar/Avatar.jsx]: Take a look at this examples of composition:
Using the avatar with attributes
```
<Avatar styles>
```
Using with children props
```
<Avatar styles>
   <Image src="/foo/bar">
<Avatar>
```
Here you have a new component `Image` and if you look at styles you already have this two components. May be you can ask. If I need an Avatar with imagem already inside? For this you can create other component and you will have a composition. e.g.
```
import {Avatar, Img} from @memed-dev/epiderme;
 
const AvatarImage = (props) => {
  return (
    <Avatar styles>
      <Image src="{props.src}">
    </Avatar>
  );
}
```

File: src/components/Avatar/README.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# Epiderme Avatar
+
+Avatars can be created by passing a img for prop `src` or without `src` for a avatar with color. 👊
+
+
+## Props
+
+| Name    | Type    | Default   | Description               | Required  |
+| ------- |:-------:| --------: | ----------:               | --------: |
+| src     | String  | -         | Patch for image/icon      | false     |
```
[the remark this answers is not in the document]
Comment (Felipesdeveloper) [reply src/components/Avatar/README.md]: I will make a change in this component, I will create an Image component that will need to pass `src`

[the remark this answers is not in the document]
Comment (Felipesdeveloper) [reply src/components/Avatar/README.md]: Man i updated the component, can you take a look again?

[the remark this answers is not in the document]
Comment (r0nho) [reply src/components/Avatar/README.md]: Nice!

**Comment by csilva2810** [comment]
:shipit:

**Pull request merged**